src/content/smart-bulletins/036.md

```diff
@@ -0,0 +1,51 @@
+
+### GSA Leased Vehicles
+-  To enable the WEX fleet card for use at ChargePoint stations, the following options are available:
+    -  For agencies with at least one owned ChargePoint station, WEX accounts can be linked to ChargePoint RFIDs in the data portal or through the ChargePoint website. This action must be taken by the agency Fleet Manager(s).
+    - For agencies/drivers that want to charge at public ChargePoint stations contact GSA Fleet’s Alternative Fuel Vehicle Team at: gsafleetafvteam@gsa.gov and provide EV tag number(s) and delivery location(s). GSA Fleet will provide the driver with ChargePoint Radio Frequency IDentification (RFID) card(s) for the vehicle(s) linked to the vehicle’s associated GSA SmartPay fleet charge card. GSA Fleet EV commercial charging information resources include: a charging stations purchasing options and charging procedures at
+[gsa.gov/evse](http://gsa.gov/evse).
+- If an agency desires to use additional commercial charging station vendors beyond ChargePoint for GSA Fleet leased EVs, it must first contact the GSA Fleet Alternative Fuel Vehicle Team at: gsafleetafvteam@gsa.gov.
+### Agency-Owned Assets
+- Agencies are not limited to the use of ChargePoint charging stations for EV commercial charging of agency-owned assets. However, as stated above, Currently, ChargePoint is one of the few commercial charging vendors at the time of the publication of this bulletin accepting WEX and Voyager. For agencies with at least one owned ChargePoint station, a Fleet account may be set up with ChargePoint through their app or website. Further information on account set up for Fleet Card Managers can be found [here](https://docs.google.com/document/d/1tm6H3aQe-JqC9qcgkbx-dxLJdkFpB2ecLsFVi0Qm6pE/edit). 
+- Voyager specific account set up information can be found on the ChargePoint site under the [Voyager Fleet Card Integration FAQs](https://www.chargepoint.com/solutions/voyager-faq/).
+- Note that most commercial charging stations do not have credit card terminals to swipe or “dip” cards, but some EVgo charges do offer charge card terminals/readers. An account will first need to be established with the commercial charging network, typically through an app or website. The GSA SmartPay card account number and related information would be needed to establish the account. Most commercial charging networks offer payments on an individual and subscription basis. Often an RFID token (“fob”) or card is required for use at the charging station to enable the transaction.
+- RFID cards/keyfobs for use at commercial EV charging stations are available on GSA Advantage or the open market where not available through GSA.
+- Agencies/organizations should be sure to confirm RFID/key fob acceptance with the desired networks before purchasing.
+### EV Rental in Travel Duty Status (TDY)
```

Comment:
EV Rental in Travel Duty Status (TDY)
If consistent with agency policy, GSA SmartPay travel charge cards may be used to pay for charging EVs rented while an employee is on TDY.
As mentioned above, the employee may need to establish an account with the charging station vendor well before their trip to receive the RFID card/fob.
Also, rental car agreements should be consulted to determine if the rental car company requires specific commercial charging networks.